File: CS/E372.Module/DatabaseUpdate/Updater.cs
using System;
using DevExpress.ExpressApp.Updating;
using DevExpress.ExpressApp;
using DevExpress.Persistent.BaseImpl;

namespace E372.Module.DatabaseUpdate {
    public class Updater : ModuleUpdater {
        public Updater(IObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            if (ObjectSpace.GetObjectsCount(typeof(Person), null) == 0) {
                Person p = ObjectSpace.CreateObject<Person>();
                p.FirstName = "Test Person";
                ObjectSpace.CommitChanges();
            }
        }
    }
}

File: VB/E372.Module/DatabaseUpdate/Updater.vb
Imports System
Imports DevExpress.ExpressApp.Updating
Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.BaseImpl

Namespace E372.Module.DatabaseUpdate
    Public Class Updater
        Inherits ModuleUpdater

        Public Sub New(ByVal objectSpace As IObjectSpace, ByVal currentDBVersion As Version)
            MyBase.New(objectSpace, currentDBVersion)
        End Sub
        Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
            MyBase.UpdateDatabaseAfterUpdateSchema()
            If ObjectSpace.GetObjectsCount(GetType(Person), Nothing) = 0 Then
                Dim p As Person = ObjectSpace.CreateObject(Of Person)()
                p.FirstName = "Test Person"
                ObjectSpace.CommitChanges()
            End If
        End Sub
    End Class
End Namespace
